Makefile

```diff
@@ -0,0 +1,12 @@
+.PHONY: help
+help: ## Displays this list of targets with descriptions
+	@echo "You prepend/append the argument 'ENV=(local|docker)' to each target. This specifies,"
+	@echo "whether to execute the target within your local environment, or docker (default).\n"
```

Comment:
This doesn't apply to this makefile, the env params are only evaluated in render-guides, the docker container doesn't utilize it.

Reply:
Ack